### PR TITLE
Update microsoft-365-compliance-center-permissions.md

### LINKED ISSUE #11651: Administrative Units are not in preview
Admin Units came out of Preview in late 2020.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 468343fc-2808-0f78-f1a7-9d34b8fa8e16
* Version Independent ID: e3b7630f-75a9-d236-6c8b-7ebd27a7e9c0
* Content: [Permissions in the Microsoft Purview compliance portal - Microsoft Purview (compliance)](https://learn.microsoft.com/en-us/microsoft-365/compliance/microsoft-365-compliance-center-permissions?view=o365-worldwide#administrative-units-preview)
* Content Source: [microsoft-365/compliance/microsoft-365-compliance-center-permissions.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/compliance/microsoft-365-compliance-center-permissions.md)
* Service: **o365-seccomp**
* GitHub Login: @robmazz
* Microsoft Alias: **robmazz**

### PR DESCRIPTION
**Pull request by msbemba**
removed preview from administrative units as it is out of preview

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/11651

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 please copy review

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: microsoft-365/compliance/microsoft-365-compliance-center-permissions.md [hunk 1]
```diff
@@ -171,10 +171,9 @@ Complete the following steps to add users or groups to a Microsoft Purview role
 7. If the selected users or groups need organization-wide access as part of this role group assignment, go to Step 10.
 8. If the selected users or groups need to be assigned to administrative units, select the users or groups and select **Assign admin units**.
 
-    > [!IMPORTANT]
-    > Administrative units are supported only in Microsoft 365 commercial cloud organizations for public preview.
 
-9. On the **Assign admin units (preview)** pane, select the checkbox for all the administrative units you want to assign to the users or groups. Select **Select**.
+
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/microsoft-365-compliance-center-permissions.md]:
```suggestion
```

File: microsoft-365/compliance/microsoft-365-compliance-center-permissions.md [hunk 2]
```diff
@@ -171,10 +171,9 @@ Complete the following steps to add users or groups to a Microsoft Purview role
 7. If the selected users or groups need organization-wide access as part of this role group assignment, go to Step 10.
 8. If the selected users or groups need to be assigned to administrative units, select the users or groups and select **Assign admin units**.
 
-    > [!IMPORTANT]
-    > Administrative units are supported only in Microsoft 365 commercial cloud organizations for public preview.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/microsoft-365-compliance-center-permissions.md:174]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@robmazz Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
Closing as per the comment in #11651

**Pull request closed without merging**